src/pmdas/farm/help

```diff
@@ -0,0 +1,705 @@
+
+@farm.ata.environment_statistics.time_in_under_temperature Under temperature time
+Time in under temperature (minutes) for the drive
+
+@farm.ata.environment_statistics.specified_max_operating_temperature Max operating temperature
+Specified maximum operating temperature for the drive
+
+@farm.ata.environment_statistics.specified_min_operating_temperature Min operating temperature
+Specified minimum operating temperature for the drive
+
+@farm.ata.environment_statistics.current_relative_humidity Relative humidity
+Current relative humidity recorded by the drive
+
+@farm.ata.environment_statistics.current_motor_power Current motor power for the drive
+@farm.ata.environment_statistics.current_12_volts Current 12 volts reading
+@farm.ata.environment_statistics.minimum_12_volts Minimum 12 volts reading
+@farm.ata.environment_statistics.maximum_12_volts Maximum 12 volts reading
+@farm.ata.environment_statistics.current_5_volts Current 5 volts reading
+@farm.ata.environment_statistics.minimum_5_volts Minimum 5 volts reading
+@farm.ata.environment_statistics.maximum_5_volts Maximum 5 volts reading
+@farm.ata.environment_statistics.power_average_12v 12 volts power average reading
+@farm.ata.environment_statistics.power_minimum_12v 12 volts power minimum reading
+@farm.ata.environment_statistics.power_maximum_12v 12 volts power maximum reading
+@farm.ata.environment_statistics.power_average_5v 5 volts power average reading
+@farm.ata.environment_statistics.power_minimum_5v 5 volts power minimum reading
+@farm.ata.environment_statistics.power_maximum_5v 5 volts power maximum reading
+
+@farm.ata.reliability_statistics.error_rate_smart_1_raw SMART 1 (Raw)
+Error rate (SMART attribute 1 Raw) for the drive
+
+@farm.ata.reliability_statistics.error_rate_smart_1_normalized SMART 1 (Normalized)
+Error rate (SMART attribute 1 Normalized) for the drive
+
+@farm.ata.reliability_statistics.error_rate_smart_1_worst SMART 1 (Worst)
+Error rate (SMART attribute 1 Worst) for the drive
+
+@farm.ata.reliability_statistics.seek_error_rate_smart_7_raw SMART 7 (Raw)
+Seek error rate (SMART attribute 7 Raw) for the drive
+
+@farm.ata.reliability_statistics.seek_error_rate_smart_7_normalized SMART 7 (Normalized)
+Seek error rate (SMART attribute 7 Normalized) for the drive
+
+@farm.ata.reliability_statistics.seek_error_rate_smart_7_worst SMART 7 (Worst)
+Seek error rate (SMART attribute 7 Worst) for the drive
+
+@farm.ata.reliability_statistics.high_priority_unload_events High priority unload events count
+Number of high priority unload events for the drive
+
+@farm.ata.reliability_statistics.helium_pressure_threshold_tripped Helium pressure threshold trip
+Helium pressure threshold tripped for the drive
+
+@farm.ata.reliability_statistics.lbas_corrected_by_parity_sector LBA corrected by parity sector 
+LBAs corrected by parity sector for the drive
+
+@farm.ata.reliability_statistics.dvga_skip_write_detected_head_0 DVGA skip write detect by head 0 for the drive
```

Comment:
As above, use an instance domain here for all the different 'head_N' instances of this metric.